lms/static/js/student_account/views/RegisterView.js

```diff
@@ -195,7 +195,7 @@
 
                             // Hide each input tip
                             $(this).children().each(function() {
-                                if (inputTipSelectors.indexOf($(this).attr('class')) >= 0) {
+                                if (inputTipSelectors.indexOf($(this).attr('class')) >= 1) {
```

Comment:
Minor nit: I would feel slightly more happy if this line had a comment for why 0 is skipped.

But :clap: :clap: on the change itself.